eventActions/afkMessageCheckAction.js

```diff
@@ -10,7 +10,7 @@ class afkMessageCheckAction {
 			return;
 		}
 		const sender = message.author;
-		const reactionFilter = (reaction, user) => {
+		(reaction, user) => {
```

Comment:
Wait, does this code block ever get run? I know you're not fixing `!afk` in this PR, but this doesn't look right. In any case, this change is fine for now.

Reply:
Yeah, doesn't look good to me either. I'll probably go back and clean up the `!afk` command in a separate PR.